servicetest/capabilities/test_caps.py

```diff
@@ -245,7 +248,7 @@ def test_set_same_cap_twice(self, agent):
                 # Set the same capability again
                 sc.set_capabilities(["test.cap.valid-dbus"])
             # The expected failure is 'Failed', not 'NoReply'
-            assert err.value.get_dbus_name() == "org.freedesktop.DBus.Error.Failed"
+            assert err.value.get_dbus_name() == sc.DBUS_EXCEPTION_FAILED
```

Comment:
I think all of these are better accessed like "Container.DBUS_EXCEPTION_FAILED" to make it clear that it's intended to be a "constant" on the class, not the object.